diff --git a/dist/components/field/field-input.vue.js b/dist/components/field/field-input.vue.js
index 40ca1525c479127c575772f43c8e84b69a681652..c91365b87414966f1250912612c18eb6f7594fcf 100644
--- a/dist/components/field/field-input.vue.js
+++ b/dist/components/field/field-input.vue.js
@@ -19,12 +19,7 @@ const _sfc_main = /* @__PURE__ */ defineComponent({
         "as-child": _ctx.asChild,
         value: _ctx.modelValue,
         onInput: _cache[0] || (_cache[0] = (event) => emit("update:modelValue", event.target.value))
-      }), {
-        default: withCtx(() => [
-          renderSlot(_ctx.$slots, "default")
-        ]),
-        _: 3
-      }, 16, ["as-child", "value"]);
+      }), undefined, 16, ["as-child", "value"]);
     };
   }
 });
diff --git a/dist/components/field/field-textarea.vue.js b/dist/components/field/field-textarea.vue.js
index d8f4fd71985e1bff86cb7df4fbe327b0463a4a95..471d5bcc65b455998f139bb1b4159c759e51f0d9 100644
--- a/dist/components/field/field-textarea.vue.js
+++ b/dist/components/field/field-textarea.vue.js
@@ -22,7 +22,7 @@ const _sfc_main = /* @__PURE__ */ defineComponent({
         value: _ctx.modelValue,
         onInput: _cache[0] || (_cache[0] = (event) => emit("update:modelValue", event.target.value)),
         style: props.autoresize ? { resize: "none", overflow: "hidden" } : void 0
-      }), "    <slot />\n  ", 16, _hoisted_1);
+      }), undefined, 16, _hoisted_1);
     };
   }
 });
