diff --git a/node_modules/@adonisjs/assembler/build/index.js b/node_modules/@adonisjs/assembler/build/index.js
index b12b4b9..69f494d 100644
--- a/node_modules/@adonisjs/assembler/build/index.js
+++ b/node_modules/@adonisjs/assembler/build/index.js
@@ -611,6 +611,13 @@ var DevServer = class {
       scriptArgs: this.#options.scriptArgs
     });
     this.#httpServer.on("message", async (message) => {
+      if (message.type === 'hot-hook:full-reload') {
+        this.#clearScreen();
+        this.#logger.log(`${this.#colors.green("full-reload")} due to ${this.#colors.cyan(message.path)}`);
+        this.#restartHTTPServer(port);
+        return;
+      }
+
       if (this.#isAdonisJSReadyMessage(message)) {
         const host = message.host === "0.0.0.0" ? "127.0.0.1" : message.host;
         const displayMessage = ui3.sticker().useColors(this.#colors).useRenderer(this.#logger.getRenderer()).add(`Server address: ${this.#colors.cyan(`http://${host}:${message.port}`)}`).add(
